packages/server/src/api/controllers/static/index.ts

```diff
@@ -167,10 +222,23 @@ export const serveApp = async function (ctx: Ctx) {
     const appInfo = await db.get<any>(DocumentType.APP_METADATA)
     let appId = context.getAppId()
 
+    const hideDevTools = !!ctx.params.appUrl
+    const sideNav = appInfo.navigation.navigation === "Left"
+    const hideFooter = ctx.user?.license?.features?.includes("branding")
+    const themeVariables = getThemeVariables(appInfo?.theme)
+
     if (!env.isJest()) {
       const plugins = objectStore.enrichPluginURLs(appInfo.usedPlugins)
+      const Skeleton =
+        require("@budibase/frontend-core/src/components/ClientAppSkeleton.svelte").default
```

Comment:
Do we need the full path? Can't we require `ClientAppSkeleton` from the index?

Reply:
Yep it's because we just have a single export of all components rather than named exports, so it's not tree-shakable and will pull in all components. Importing it from the path directly here to save on bloat is a good idea 👍